dashboard/lib/services/i18n/curriculum_sync_utils/serializers.rb

```diff
@@ -124,38 +124,6 @@ class ObjectiveCrowdinSerializer < CrowdinSerializer
       attribute :description
     end
 
-    class FrameworkCrowdinSerializer < CrowdinSerializer
-      attributes :name
-
-      delegate :crowdin_key, to: :object
-    end
-
-    class StandardCategoryCrowdinSerializer < CrowdinSerializer
-      attributes :description
-
-      delegate :crowdin_key, to: :object
-    end
-
-    class StandardCrowdinSerializer < CrowdinSerializer
-      attributes :description
-
-      belongs_to :framework, serializer: FrameworkCrowdinSerializer, optional: true
-      belongs_to :parent_category, serializer: StandardCategoryCrowdinSerializer, optional: true
-      belongs_to :category, serializer: StandardCategoryCrowdinSerializer, optional: true
-
-      delegate :crowdin_key, to: :object
-    end
-
-    class ReferenceGuideCrowdinSerializer < CrowdinSerializer
```

Comment:
Looks like you accidentally removed this serializer. This was what I added with a merged PR this morning, so maybe it had to do with sequencing of merging with staging?